File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.c
int removeDuplicates(int* nums, int numsSize) {
    int j=0;
    for (int i=1;i<numsSize;i++){
        if (nums[i]!=nums[j]){
            j++;
            nums[j]=nums[i];
        }
    }
    return j+1;
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i=0
        while(i<len(nums)-1):
            n,m=nums[i],nums[i+1]
            if n!=m:
                i+=1
            else:
                nums.pop(i)
        return i+1